index.js

```diff
@@ -110,7 +114,10 @@ function formatWithCursor(text, opts, addAlignmentSize) {
   const doc = printAstToDoc(ast, opts, addAlignmentSize);
   opts.newLine = guessLineEnding(text);
   const toStringResult = printDocToString(doc, opts);
-  const str = toStringResult.formatted;
+  let str = toStringResult.formatted;
+  if (hasUnicodeBOM) {
+    str = "\ufeff" + str;
```

Comment:
I think we should use `String.fromCharCode(UTF8BOM)` here instead of `"\ufeff"`. Other than that, this looks good to me!

Reply:
@josephfrazier  done. Thanks.